packages/wallet/backend/src/rafiki/service.ts

```diff
@@ -315,7 +317,8 @@ export class RafikiService implements IRafikiService {
     )
 
     const user = await this.deps.userService.getByWalletId(source_ewallet)
-    if (user)
+    const isExchange = Node_Cache.get(wh.data.payment.id)
```

Comment:
is the payment id equal with quote id?

Reply:
It's a little weird, but according to logs and database records, the IDs of outgoing payments and quotes are the same.